Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -53,6 +54,84 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
             });
         },
 
+        /**
+         * Fetches data about user
+         *
+         * @method _fetchUserInfo
+         * @param event {Object} event facade
+         */
+        _fetchUserInfo: function (event) {
+            var user = this.get('app').get('user'),
+                userService = this.get('capi').getUserService(),
+                userInfo = {},
+                promise = new Y.Promise(function (resolve, reject) {
+                    userService.loadUser(user.get('id'), Y.bind(function (error, response) {
```

Comment:
this part is useless, the currently logged in user is already loaded and available in the `user` attribute of the application. (`this.get('app').get('user');` from here).